server/src-lib/Hasura/Server/App.hs

```diff
@@ -557,13 +573,17 @@ httpApp corsCfg serverCtx enableConsole consoleAssetsDir enableTelemetry = do
       post "v1alpha1/graphql/explain" gqlExplainAction
 
       post "v1alpha1/graphql" $ mkSpockAction GH.encodeGQErr id serverCtx $
-        mkPostHandler $ mkAPIRespHandler v1Alpha1GQHandler
+        mkPostHandler $ mkAPIRespHandler $ v1Alpha1GQHandler pure
 
       post "v1/graphql/explain" gqlExplainAction
 
       post "v1/graphql" $ mkSpockAction GH.encodeGQErr allMod200 serverCtx $
         mkPostHandler $ mkAPIRespHandler v1GQHandler
 
+      post ("v1/graphql/remote" <//> var) $ \(RemoteSchemaName -> remoteSchemaName) ->
```

Comment:
This is needed currently for the console to show the remote schema types in the 'remote schema permissions' window. So to fetch the types, console will hit the `graphql/remote/<remote-name>` endpoint with an introspection query. 

I'm wondering regardless of this remote schema proxy endpoint, if it is good to have something like `execute_graphql_query` in `v1/query` or even scope it to something like `fetch_remote_schema_types`?

Reply:
I guess its fine to create a metadata api like `introspect_remote_schema` but I don't think an API like `execute_graphql_query` is much usable as it won't benefit from any graphql client side tooling.